code/modules/mob/mob.dm

```diff
@@ -307,6 +307,70 @@
 	face_atom(A)
 	A.examine(src)
 
+	
+/mob/living/carbon/verb/give()
+	set category = "IC"
+	set name = "Give"
+	set src in view(1)
+
+	if(src == usr)
+		to_chat(usr,"<span class='warning'>You feel stupider, suddenly.</span>")
+		return
+
+	if(!ismonkey(src)&&!ishuman(src) || isalien(src) || src.stat || usr.stat || !src.client)
```

Comment:
if something is not a monkey or not a human, then why are you checking seperately if its an alien. the alien would be in the "not a monkey and not a human" category.